api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -8580,19 +9045,83 @@ public CohortDefinition getMQ15DPatientsMarkedAsBreastfeeding() {
    *
    *
    * <ul>
-   *   <li>B4_2 - Filter all patients with a “’Ultima Profilaxia Isoniazida (Data Início)” (concept
-   *       id 6128) in Ficha Resumo (encounter type 53) during the Inclusion period as following:
+   *   <li>B4_1 - Filter all patients with “Ultima Profilaxia TPT (concept_id 23985) = INH (concept
+   *       = 656) and (Data Início)” (obs_datetime for concept id 165308 value 1256) in Ficha Resumo
+   *       (encounter type 53) during the Inclusion period as following:
```

Comment:
@TinoMuianga can you change the spec to what RF12 says?

Reply:
@psmatsinhe It is done. The thing with the requirement document is that the specification is limited, on the description of the actual concepts being used.